### DISCUSSION TITLE
Piano LED Visualizer without MIDI Interface - oldschool Analog Piano

### DISCUSSION THREAD

**Discussion by Peer256** [onlaj/Piano-LED-Visualizer#264] [Q&A] [answered]
Hi everyone,

we want to built this project, but without a E-Piano simply with an oldschool one. Does the Visualizer requiere a MIDI-Interface to work or is it possible to just visualise the MIDI files onto the LEDs?

Thanks in advance!

Peer

**Comment by onlaj** [comment, marked as answer]
Hello.
You can play midi files using built-in midi player, it will light up corresponding leds.
